client/src/pages/lessons/edit.tsx

```diff
@@ -212,6 +280,128 @@ const LessonEdit = (props: {
   } = useWithDownload(lessonId, context.user, cookies.accessToken);
 
   const [mounted, setMounted] = React.useState(false);
+
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+  const [selectedIndex, setSelectedIndex] = React.useState(0);
+  const open = Boolean(anchorEl);
+  const handleClickListItem = (event: React.MouseEvent<HTMLElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+  const drawerWidth = 240;
+
+  const openedMixin = (theme: Theme): CSSObject => ({
+    width: drawerWidth,
+    transition: theme.transitions.create("width", {
+      easing: theme.transitions.easing.sharp,
+      duration: theme.transitions.duration.enteringScreen,
+    }),
+    overflowX: "hidden",
+  });
+
+  const closedMixin = (theme: Theme): CSSObject => ({
```

Comment:
again I'm not sure about the best practices regarding CSS,  It seems to me that the CSS should be defined separately from the functionality.